src/charm.py

```diff
@@ -59,6 +59,25 @@ def _on_config_changed(self, event):
     def _install_dependencies(self):
         apt.update()
         apt.add_package(REQUIRED_APT_PACKAGES)
+        if self.config['mount_type'] == "efs":
```

Comment:
Can we check that `mount_type` has a valid value? Currently if `mount_type != efs` we always assume `nfs` and also forward the value without further checking to the systemd unit below.